x-pack/plugins/data_visualizer/public/application/index_data_visualizer/hooks/esql/use_esql_overall_stats_data.ts

```diff
@@ -91,7 +91,8 @@ const getESQLDocumentCountStats = async (
       ` | EVAL _timestamp_= TO_DOUBLE(DATE_TRUNC(${intervalMs} millisecond, ${getSafeESQLName(
         timeFieldName
       )}))
-    | stats rows = count(*) by _timestamp_`
+    | stats rows = count(*) by _timestamp_
+    | LIMIT 1000`
```

Comment:
Let me take a look what would be a good limit for this query. 1000 buckets might be too low.

Reply:
Actually, this is good since that's what Elasticsearch defaults to anyway. Thanks!